content/v2.0/architecture/other-files.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Architecture: Overview"
+order: 20
+---
+
+## Anatomy Of A Project
+
+We have already examined `lib/` and `slices/`, but there are other parts of a newly generated project that deserve to be explained.
+
+```shell
+tree -L 1                                                                      
+.
+├── Brewfile
+├── Brewfile.lock.json
```

Comment:
Seems this file isn't existing in a newly created hanami 2 application (currently through the template repo).

Reply:
In beta even more files had been removed. This will need another walkthrough.